detectors/yolo/yolo_detector.py

```diff
@@ -1,10 +1,28 @@
 import cv2
 import numpy as np
 import os
+from configparser import ConfigParser
 
+class VehicleCountingConfigNotFound(Exception):
```

Comment:
Trying to wrap my head around why there are so many empty custom exceptions. I'm guessing the changes made to this file are intended to validate custom paths for a YOLO weights and config file. But it's overly complex. Also seems needless. Without it, you still get an error if the file paths are wrong. If I understand the tests correctly, they are checking that the weights and config are in the correct directory. Not necessary at all IMHO. I realize tests are important and have plans to write a comprehensive test suite for this project so I'm not against testing, but these tests are not needed here right now. I recommend you remove these changes from the PR.

Reply:
I only added them to make the test work, I would have preferred just testing the error message, I can't seem to get that to work on the fly. So, I agree that we should keep it in view for now and focus on getting things to work. 

But what do you think about the config idea though ?

Reply:
Having a config file is good, but I think it can makes things a wee bit complicated. I feel that for now, dropping your weights and config files in a specified folder is the easiest way to go, rather than putting it where you like and having to tinker with file path names and all.